applications/OptimizationApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -126,19 +127,56 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
     AddContainerVariableDataHolderTypeToPython<ModelPart::ConditionsContainerType, PropertiesContainerDataIO>(m, "ConditionPropertiesContainerVariableDataHolder");
     AddContainerVariableDataHolderTypeToPython<ModelPart::ElementsContainerType, PropertiesContainerDataIO>(m, "ElementPropertiesContainerVariableDataHolder");
 
+    py::class_<CollectiveVariableDataHolder, CollectiveVariableDataHolder::Pointer>(m, "CollectiveVariableDataHolder")
+        .def(py::init<>())
+        .def(py::init<const CollectiveVariableDataHolder&>())
+        .def(py::init<const std::vector<CollectiveVariableDataHolder::ContainerVariableDataHolderPointerVariantType>&>())
+        .def("AddVariableDataHolder", &CollectiveVariableDataHolder::AddVariableDataHolder)
+        .def("AddCollectiveVariableDataHolder", &CollectiveVariableDataHolder::AddCollectiveVariableDataHolder)
+        .def("ClearVariableDataHolders", &CollectiveVariableDataHolder::ClearVariableDataHolders)
+        .def("GetVariableDataHolders", py::overload_cast<>(&CollectiveVariableDataHolder::GetVariableDataHolders))
+        .def("IsCompatibleWith", &CollectiveVariableDataHolder::IsCompatibleWith)
+        .def("Clone", &CollectiveVariableDataHolder::Clone)
+        .def("CloneWithDataInitializedToZero", &CollectiveVariableDataHolder::CloneWithDataInitializedToZero)
+        .def(py::self +  py::self)
+        .def(py::self += py::self)
+        .def(py::self +  float())
+        .def(py::self += float())
+        .def(py::self -  py::self)
+        .def(py::self -= py::self)
+        .def(py::self -  float())
+        .def(py::self -= float())
+        .def(py::self *  py::self)
+        .def(py::self *= py::self)
+        .def(py::self *  float())
+        .def(py::self *= float())
+        .def(py::self /  py::self)
+        .def(py::self /= py::self)
+        .def(py::self /  float())
+        .def(py::self /= float())
```

Comment:
I'll go ahead and repeat my concerns about these operations from the previous PR.

Reply:
It is required, but done as in the `KratosCore::ContainerExpression`.